### ISSUE TITLE
[Feature request] "Just boot" option

### ISSUE THREAD

**Issue by OatmealDome** [checkra1n/BugTracker#110]
For those who just want to apply custom boot-args and/or kernel patches, and skip "fully jailbreaking" their device and installing a package manager.

For example: ``checkra1n_gui -b "-v"`` would verbose boot the device.

**Comment by Lonegwadiwaitor** [comment]
This is for bugs, not for feature suggestions.

**Comment by OatmealDome** [comment]
@Lonegwadiwaitor 

1. Where else should I put this then?

2. There is a "free form" issue template for "everything that is not a bug report". I think feature requests fit into this category.

**Comment by woachk** [comment]
@OatmealDome We currently don't have a way to pass arguments.

Skip ["fully jailbreaking" their device and] installing a package manager is what's done already, Cydia isn't installed by default for example and can be installed via the on-device checkra1n app.

**Comment by OatmealDome** [comment]
@woachk Is it possible this can be implemented in the future? Having minimal changes applied to the device (ie no "loader.app" installed, "checkra1nfs" disk image contents not copied/overlayed onto rootfs) would be optimal for me.

**Comment by nullpixel** [comment]
checkra1nfs is mounted read only to / and is never copied. loader.app is copied to a temporary folder in var but never to the rootfs.

**Comment by Siguza** [comment]
I'm putting this on the list of "TODOs for the distant future". We'll get back to this once we worked through our currently planned feature set.

**Comment by alessaba** [comment]
I agree with @nullpixel on this one. If one just wants to set custom args (future feature), verbose boot or apply custom kernel patches, you can already do it without installing any package manager. 

As long as one doesn’t install a package manager, checkra1n isn’t a conventional user jailbreak in itself. It just does patches, and gives a local shell. There is no permanent file system change, it's all temporary.

The only change that maybe the devs could do is just give the option not to install the Loader app, but you can just as well drag the Loader into a homescreen folder and call it a day

**Comment by nullpixel** [comment]
> The only change that maybe the devs could do is just give the option not to install the Loader app, but you can just as well drag the Loader into a homescreen folder and call it a day

It doesn't even show on reboot, because it's only stored in a temp place!

**Comment by alessaba** [comment]
Wait, it's been a long time since i've booted unjailbroken. Does the loader disappear when in unjailbroken mode, too? I thought it stayed on the springboard but crashed on open (like cydia)

**Comment by alessaba** [comment]
Yeah, that’s true
